UserGuide.md

```diff
@@ -143,7 +146,7 @@ The relationship between **pipelines, templates, tests, runs**, and **results**
 
 ![Test Structure](images/Test_Structure.png)
 
-The following sections detail some specifics for how to use CARROT to create and dispatch analyses.  These sections refer to “tests” largely as notional concepts, rather than the CARROT **test** defined above.  In the future the terminology will be updated to be less ambiguous / confusing.
+The following sections detail some specifics for how to use CARROT to create and dispatch analyses.  These sections refer to “tests” largely as notional concepts, rather than the CARROT **test** defined above.
```

Comment:
Kind of confusing to define test and then say "well, we're going to ignore that."  I'm not sure "notional concepts" is what you mean.  (Unless you mean tests are imaginary concepts.)

Reply:
I'll try to clarify.  Admittedly having an entity named "test" makes things slightly confusing.